ligtr=/scr/drew/lig_train1.molcache2, rectr=/scr/drew/rec_train1.molcache2
DDG_reg.py:61: UserWarning: Using a target size (torch.Size([42])) that is different to the input size (torch.Size([42, 1])). This will likely lead to incorrect results due to broadcasting. Please ensure they have the same size.
  loss = F.mse_loss(output,labels)
Traceback (most recent call last):
  File "DDG_reg.py", line 131, in <module>
    tr_loss, tr_r2 =  train(config, model, traine, optimizer, epoch)
  File "DDG_reg.py", line 61, in train
    loss = F.mse_loss(output,labels)
  File "/net/mahler/home/mscbio/anm329/.local/lib/python3.6/site-packages/torch/nn/functional.py", line 2204, in mse_loss
    ret = torch._C._nn.mse_loss(expanded_input, expanded_target, _Reduction.get_enum(reduction))
RuntimeError: Expected object of scalar type Float but got scalar type Double for argument #2 'target' in call to _thnn_mse_loss_forward
